packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -53,7 +55,7 @@ exports.onPostBuild = (args, pluginOptions) => {
         `${process.cwd()}/${rootDir}/offline-plugin-app-shell-fallback/index.html`
       )
     )
-  )
+  ).map(omitPrefix)
```

Comment:
Not sure if this is necessary with the change on line 79 but I'll just do some of my own testing to see what happens